### PR TITLE
type bug fixes

### PR DESCRIPTION
**Pull request by jineshshah36**

### CODE REVIEW

**Comment by jineshshah36** [comment]
Just fixing a few type issues I noticed

File: index.d.ts [hunk 1]
```diff
@@ -1367,14 +1367,44 @@ declare namespace Shopify {
...
+        product_id: number;
+        requires_shipping: boolean;
+        taxable: boolean;
+        gift_card: boolean;
+        name: string;
+        variant_inventory_management: string;
+        product_exists: boolean;
+        fulfillable_quantity: number;
+        total_discount: string;
+        fulfillment_status: null;
```
Comment (lpinca) [inline index.d.ts]: This can be `'fulfilled'`, `null` or `'partial'`.

File: index.d.ts [hunk 2]
```diff
@@ -1367,14 +1367,44 @@ declare namespace Shopify {
...
+        variant_title: string;
+        vendor: string | null;
+        fulfillment_service: string;
+        product_id: number;
+        requires_shipping: boolean;
+        taxable: boolean;
+        gift_card: boolean;
+        name: string;
+        variant_inventory_management: string;
+        product_exists: boolean;
```
Comment (lpinca) [inline index.d.ts]: The `properties` property is missing.

Comment (jineshshah36) [reply index.d.ts]: Fixed by https://github.com/MONEI/Shopify-api-node/pull/250/commits/a877d03b6d964e6ec02d7e8043efa887958d232d

File: index.d.ts (hunk 1 above)
Comment (jineshshah36) [reply index.d.ts]: Fixed by https://github.com/MONEI/Shopify-api-node/pull/250/commits/a877d03b6d964e6ec02d7e8043efa887958d232d

**Comment by lpinca** [comment]
Thank you.

**Comment by jineshshah36** [comment]
At what cadence do you release to npm?

**Comment by lpinca** [comment]
There isn't a predefined schedule. I can cut a new version later today if you rely on these fixes.

**Comment by jineshshah36** [comment]
No big deal, I'm rely on my patched branch temporarily, but just want to be on the lookout

**Comment by lpinca** [comment]
I've just published version 2.17.1

**Pull request merged**